webapp-django/crashstats/crashstats/templates/crashstats/topcrasher.html

```diff
@@ -33,6 +33,9 @@
                   {% endif %}
                 {% endif %}
             </ul>
+            <a href="{{ url('topcrashers:topcrashers') }}?product={{ product }}&amp;version={{ versions[0] }}&amp;date_range_type={{ date_range_type }}&amp;process_type={{ crash_type.replace('all', 'any') }}&amp;platform={{ os_name }}&amp;_facets_size={{ result_count }}&amp;days={{ days }}">
+                See the new Top Crashers report
```

Comment:
Up to you. You don't have to agree. I think I spoke mainly from my gut. Since we don't have any hard rules on these kinds of things I can't state a firm opinion. 
What do you think looks best?